clang/docs/ReleaseNotes.rst

```diff
@@ -229,6 +229,10 @@ C++20 Feature Support
   will now work.
   (#GH62925).
 
+- Clang refactored the BMI format to make it possible to support no transitive changes
```

Comment:
Maybe explicitly mention the user-facing effect of what "no transitive change" means and mention that we want feedback on this?

Something like:
- Clang implemented improvements to BMI of C++20 Modules that should reduce the number of rebuilds during incremental recompilation. We are seeking feedback from Build System authors and other interested users, especially when you feel Clang changes the BMI and missses an opportunity to avoid recompilations or causes correctness issues. See `StandardCPlusPlusModules <StandardCPlusPlusModules.html>`_ for more details. 

Also, maybe put this at the top to increase chances of people noticing this? (if we do want people filing bugs about it, it'd help to increase the chances that they do).

Reply:
Good suggestions. Applied. (Just change `should` to `can`. Since I feel `should` read like users can get this benefit directly. But it is not the case. They need helps from build systems).

And I put this into the `What's New in Clang |release|?` section, which is highest except the `Potentially Breaking Change` section.

Reply:
> Good suggestions. Applied. (Just change should to can. Since I feel should read like users can get this benefit directly. But it is not the case. They need helps from build systems).

Yeah, makse sense, "can" fits better here, we also want some follow up from the build system vendors.

> And I put this into the What's New in Clang |release|? section, which is highest except the Potentially Breaking Change section.

This seems very reasonable, I am not sure this is a "potentially breaking change" even. It does change the BMI, but that's expected with every new version of the compiler.